community/bolt/src/test/resources/authorization/auth

```diff
@@ -0,0 +1 @@
+neo4j:SHA-256,D569C062EF5D3D5A6D214BA45911329AB59840330619F47ED56BBE53032B38F3,F05C2392D0861F6BBB72B708EDFCF8CF:
```

Comment:
It is used in tests to set up an existing user that doesn't have an expired password